plugins/package-managers/conan/src/main/kotlin/Conan.kt

```diff
@@ -77,8 +78,15 @@ internal class ConanCommand(private val useConan2: Boolean = false) : CommandLin
 
     override fun getVersionRequirement(): RangesList = RangesListFactory.create(">=1.44.0 <3.0")
 
-    override fun run(vararg args: CharSequence, workingDir: File?, environment: Map<String, String>) =
-        super.run(args = args, workingDir, environment + ("CONAN_NON_INTERACTIVE" to "1"))
+    override fun run(vararg args: CharSequence, workingDir: File?, environment: Map<String, String>): ProcessCapture =
+        super.run(
+            args = args,
+            workingDir,
+            environment + mapOf(
+                "CONAN_NON_INTERACTIVE" to "1",
+                "CONAN_SERIES" to if (useConan2) "2" else "1"
```

Comment:
Can we rename `CONAN_SERIES` to e.g. `CONAN_MAJOR_VERSION` or so? "Series" sounds a bit uncommon to me.